pkg/storage/db.go

```diff
@@ -536,140 +299,26 @@ func resIDFromARN(ARN string) string {
-		}
-	}
-
-	for _, val := range changes.PrivateIPAddresses {
-		isJoin := false
-		if strings.EqualFold(added, changes.ChangeType) {
-			isJoin = true
-		}
-		err := db.insertIPAddress(ctx, val, tx)
-		if err != nil {
-			return err
-		}
-		err = db.insertNetworkChangeEvent(ctx, timestamp, false, isJoin, resourceID, val, nil, tx)
-		if err != nil {
-			return err
-		}
-	}
-
-	for _, val := range changes.PublicIPAddresses {
-		isJoin := false
-		if strings.EqualFold(added, changes.ChangeType) {
-			isJoin = true
-		}
-		err := db.insertIPAddress(ctx, val, tx)
-		if err != nil {
-			return err
-		}
-		// yeah, a nested for loop to establish the relationship of every public IP to every hostname
-		for _, hostname := range changes.Hostnames {
-			err := db.insertNetworkChangeEvent(ctx, timestamp, true, isJoin, resourceID, val, &hostname, tx)
-			if err != nil {
-				return err
-			}
-		}
-	}
-
-	return nil
-}
-
-func (db *DB) insertIPAddress(ctx context.Context, ipAddress string, tx *sql.Tx) error {
-
-	// this lib won't give back the last INSERTed row ID, so we don't bother with `RETURNING ...`
-	// See https://stackoverflow.com/questions/34708509/how-to-use-returning-with-on-conflict-in-postgresql
-	if _, err := tx.ExecContext(ctx, fmt.Sprintf(`INSERT INTO %s VALUES ($1) ON CONFLICT DO NOTHING`, tableAWSIPS), ipAddress); err != nil { // nolint
-		return err
-	}
-
-	return nil
-}
-
-func (db *DB) insertHostname(ctx context.Context, hostname string, tx *sql.Tx) error {
-
-	// this lib won't give back the last INSERTed row ID, so we don't bother with `RETURNING ...`
-	// See https://stackoverflow.com/questions/34708509/how-to-use-returning-with-on-conflict-in-postgresql
-	if _, err := tx.ExecContext(ctx, fmt.Sprintf(`INSERT INTO %s VALUES ($1) ON CONFLICT DO NOTHING`, tableAWSHostnames), hostname); err != nil { // nolint
-		return err
-	}
-
-	return nil
-}
-
-func (db *DB) insertNetworkChangeEvent(ctx context.Context, timestamp time.Time, isPublic bool, isJoin bool, resourceID string, ipAddress string, hostname *string, tx *sql.Tx) error {
-	// Postgres 11 automatically propagates the parent index to child tables, so no need to
-	// explicitly create an index on the possibly created new table.
-
-	// this lib won't give back the last INSERTed row ID, so we don't bother with `RETURNING ...`
-	// See https://stackoverflow.com/questions/34708509/how-to-use-returning-with-on-conflict-in-postgresql
-	_, err := tx.ExecContext(ctx, fmt.Sprintf(`INSERT INTO %s VALUES ($1, $2, $3, $4, $5, $6)`, tableAWSEventsIPSHostnames), timestamp, isPublic, isJoin, resourceID, ipAddress, hostname) // nolint
-	return err
-}
-
 // FetchAll gets all the assets present at the specified time
 func (db *DB) FetchAll(ctx context.Context, when time.Time, count uint, offset uint, typeFilter string) ([]domain.CloudAssetDetails, error) {
-	return db.runQuery(ctx, bulkResourcesQuery, when, typeFilter, count, offset)
+	return nil, errors.New("bulk export API is not available")
```

Comment:
Question/Nit: Is this because the bulk query relied on the old schema layout? 

In terms of nit, I think we've got a Pollinator check that hits the bulk endpoint (I'm not sure if we own it though) from our Splunk logs. If we're going to be turning the endpoint off like this it'd be nice to have a followup to stop pinging the endpoint. 

Also I see that the bulkResourcesQuery is still in the const list at the top of db.go. I think it should be taken out if this was the only place that used it.

Reply:
It is *not* supposed to work (at least it does not return any sensible results as dual-writes stopped long time ago). If there is pollinator check - we should stop it. 
We are about to re-write this code path to use new schema.
re: Query - I will drop it, thank you!

Reply:
re: Not supposed to work - it has such poor performance that running it can result in an outage. We should have added code to error immediately long ago.